frontend/src/metabase/core/components/EditableText/EditableText.tsx

```diff
@@ -96,24 +124,39 @@ const EditableText = forwardRef(function EditableText(
     [submitValue, isMultiline],
   );
 
+  const handleRootElementClick = (event: MouseEvent) => {
+    if ((event.target as HTMLElement).tagName.toLowerCase() !== "a") {
+      setIsInFocus(true);
+    }
+  };
+
+  const shouldShowMarkdown = isMarkdown && !isInFocus && inputValue;
+  const shouldShowInput = !shouldShowMarkdown;
```

Comment:
nit: I'd rather remove this check and change the code below to something like:
```
{shouldShowMarkdown ? <Markdown ... /> : <EditableTextArea ... />
```

so it's clear we either show `Markdown` or `TextArea`, not both

Reply:
yeah, there were a couple of iterations on refactoring on this code, good point